infrastructure/environments/dplplat01/configuration/mariadb-servers/templates/mariadb-server.template.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.namespace }}
```

Comment:
You don't need to create the namespace here. Just install the chart with --create-namespace, and it will be created on install.

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/632/commits/7f5c27305b69324c12f2603c53ac6ce8e85c137c